app/models/order_cycle.rb

```diff
@@ -281,7 +281,9 @@ def orders_close_at_after_orders_open_at?
 
   def reset_processed_at
     return unless orders_close_at.present? && orders_close_at_was.present?
+    return unless orders_close_at > orders_close_at_was
 
-    self.processed_at = nil if orders_close_at > orders_close_at_was
+    self.processed_at = nil
+    self.mails_sent = false
```

Comment:
If we discover more things like this then "cloning" is probably not the right approach. We could just copy the data we need to a new OC. Out of scope here.